docs/other/install-gwms-frontend.md

```diff
@@ -609,28 +623,29 @@ groupwould only match jobs that have the `+is_itb=True` ClassAd.
         <job query_expr="(!isUndefined(is_itb) && is_itb)">
         
 
-6. Reconfigure the Frontend:
+6. Reconfigure the Frontend (see the section below):
 
         :::console
-        /etc/init.d/gwms-frontend reconfig
+        # on EL7 systems
+        systemctl reload gwms-frontend
+        # on EL6 systems
+        service gwms-frontend reconfig
+        
         
-
-
 Using GlideinWMS
 ----------------
 
 ### Managing GlideinWMS Services ###
 
 In addition to the GlideinWMS service itself, there are a number of supporting services in your installation. The specific services are:
 
-| Software   | Service name                                                   | Notes                                                                        |
-|:-----------|:---------------------------------------------------------------|:-----------------------------------------------------------------------------|
-| Fetch CRL  | `fetch-crl-boot` and `fetch-crl-cron`                          | See [CA documentation](/common/ca#managing-fetch-crl-services) for more info |
-| Gratia     | `gratia-probes-cron`                                           | Accounting software                                                          |
-| HTCondor   | `condor`                                                       |                                                                              |
-| HTTPD      | `httpd`                                                        | GlideinWMS monitoring                                                        |
-| GlideinWMS | `gwms-renew-proxies` (EL6) or `gwms-renew-proxies.timer` (EL7) | [Automatic proxy renewal](#proxy-configuration)                              |
-|            | `gwms-frontend`                                                | The main GlideinWMS service                                                  |
+| Software   | Service name                             | Notes                                                                        |
+|:-----------|:-----------------------------------------|:-----------------------------------------------------------------------------|
+| Fetch CRL  | `fetch-crl-boot` and `fetch-crl-cron`    | See [CA documentation](/common/ca#managing-fetch-crl-services) for more info |
+| Gratia     | `gratia-probes-cron`                     | Accounting software                                                          |
+| HTCondor   | `condor`                                 |                                                                              |
+| HTTPD      | `httpd`                                  | GlideinWMS monitoring and staging                                            |
+| GlideinWMS | `gwms-renew-proxies` and `gwms-frontend` | [Automatic proxy renewal](#proxy-configuration) and main GlideinWMS service  |
```

Comment:
Why change this here? We need to differentiate between the EL6 and EL7 for the proxy renewal service

Reply:
In this table, you just list the different GlideinWMS services. Below, you have the list of how to enable them, either for el6 or el7.

Would you like to have it in other way?

Reply:
Yea, the thing is on EL7 you have to enable `gwms-renew-proxies.timer` and for EL6 you have to enable `gwms-renew-proxies`. The instructions below go into which commands to run on EL6 vs EL7 to do this but not the service names

Reply:
Indeed! ok, would you prefer to point it out here or in "Proxy configuration section"?

Reply:
I think here makes sense. If you just revert your change I think that should suffice

Reply:
Oh! I think I know what's happened.. This was not-intentionally change when I was fixing the merge conflicts. I'll revert the change.